portal_objects/metaworkflows/WGS_SNV_germline_proband_hg19_vcf.yaml

```diff
@@ -0,0 +1,652 @@
+#     General information for the pipeline
+#############################################################
+name: WGS_SNV_germline_proband_hg19_vcf
+description: Downstream pipeline to call SNV and small Indel for Whole Genome Sequencing, Proband. |
+             The pipeline requires an hg19/GRCh37 based VCF file as input and lifts it over to hg38/GRCh38. |  
+             Based on GATK.
+
+## General arguments ########################################
+#     Pipeline input, reference files, and general arguments
+#       define all arguments for the pipeline here
+#############################################################
+input:
+
+  # File argument
+  input_snv_vcfs:
+    argument_type: file.vcf_gz
+    dimensionality: 1
+
+  reference_fa:
+    argument_type: file.fa
+    files:
+      - complete-reference-fasta@hg38
+
+  regions:
+    argument_type: file.txt
+    files:
+      - 10Mb-genomic-regions@hg38
+
+  vep:
+    argument_type: file.vep_tar
+    files:
+      - vep@101b
+
+  clinvar:
+    argument_type: file.vcf_gz
+    files:
+      - clinvar@20201101
+
+  dbnsfp:
+    argument_type: file.dbnsfp_gz
+    files:
+      - dbnsfp@4.1a
+
+  maxent:
+    argument_type: file.tar_gz
+    files:
+      - maxent@20040421
+
+  spliceai_snv:
+    argument_type: file.vcf_gz
+    files:
+      - spliceai-snv@1.3
+
+  spliceai_indel:
+    argument_type: file.vcf_gz
+    files:
+      - spliceai-indel@1.3
+
+  gnomad:
+    argument_type: file.vcf_gz
+    files:
+      - gnomad-genome@3.1
+
+  gnomad2:
+    argument_type: file.vcf_gz
+    files:
+      - gnomad-exome@2.1.1
+
+  CADD_snv:
+    argument_type: file.tsv_gz
+    files:
+      - cadd-snv@1.6
+
+  CADD_indel:
+    argument_type: file.tsv_gz
+    files:
+      - cadd-indel@1.6
+
+  phylop100bw:
+    argument_type: file.bigWig
+    files:
+      - phylop100way-vertebrate@hg38
+
+  phylop30bw:
+    argument_type: file.bigWig
+    files:
+      - phylop30way-mammalian@hg38
+
+  phastc100bw:
+    argument_type: file.bigWig
+    files:
+      - phastcons100way-vertebrate@hg38
+
+  genes:
+    argument_type: file.txt
+    files:
+      - genes-list-ensid@0.4.6
+
+  dbSNP_full_ref_vcf:
+    argument_type: file.vcf_gz
+    files:
+      - dbsnp-all@151
+
+  chainfile_hg38_hg19:
+    argument_type: file.chain
+    files:
+      - liftover-chain@hg38to19
+  
+  chainfile_hg19_hg38:
+    argument_type: file.chain
+    files:
+      - liftover-chain@hg19to38
+
+  # Parameter argument
+  sample_names:
+    argument_type: parameter.json
+
+  pedigree:
+    argument_type: parameter.string
+
+  variant_filtering_aftag:
+    argument_type: parameter.string
+    value: gnomADg_AF
+
+  variant_filtering_afthr:
+    argument_type: parameter.float
+    value: "0.01"
+
+## Workflows and dependencies ###############################
+#     Information for the workflows and their dependencies
+#############################################################
```

Comment:
did you test this? this block is repeated once but empty, I guess it worked because the second block override the empty one. We can remove this duplicated part